mscore/file.cpp

```diff
@@ -2143,10 +2143,18 @@ bool MuseScore::savePdf(Score* cs_, QPrinter& printer)
       const QList<Page*> pl = cs_->pages();
       int pages = pl.size();
       bool firstPage = true;
+      bool usePageColor = preferences.getBool(PREF_UI_CANVAS_FG_USECOLOR_IN_EXPORT);
       for (int n = 0; n < pages; ++n) {
             if (!firstPage)
                   printer.newPage();
             firstPage = false;
+
+            if (usePageColor) {
```

Comment:
I sense a code smell here. The code itself is fine, but the fact that it's is copied and pasted two more times is not (as it contradicts the [DRY](https://de.wikipedia.org/wiki/Don%E2%80%99t_repeat_yourself) principle). 

My approach would be to move your code into a private subroutine (i.e. something like `applyPageColor(Printer, pageNum)`) and just call that.

But i can see why that that happened: At first glance it looks to me as if `savePdf(Score* cs_, QPrinter& printer)` and `savePdf(QList<Score*> cs_, const QString& saveName)` already have a serious amount of code duplication (if i'm not misunderstanding something here and they do something completely), i'd advise to refactor this at some point in the near future to reduce technical debt and make the code more stable and easier to maintain in the future.

Reply:
Taken care of :-)